danspeech/DanSpeechRecognizer.py

```diff
@@ -12,6 +13,7 @@ def __init__(self, model_name=None, lm_name=None,
                  beam_width=64):
 
         self.device = torch.device("cuda" if with_gpu else "cpu")
+        print(self.device)
```

Comment:
printing a plain number isn't all that informative, consider adding a meaningful text as well.

Reply:
Will also be cleaned! :)